Src/VsVimShared/HostFactory.cs

```diff
@@ -114,6 +118,50 @@ private void BeginSettingSynchronization(IVimBuffer vimBuffer)
             _editorToSettingSynchronizer.StartSynchronizing(vimBuffer, settingSyncSource);
         }
 
+        private void EnableVimLineNumbersMargin(IEditorOptions textViewOptions)
+        {
+            textViewOptions.SetOptionValue(DefaultTextViewHostOptions.LineNumberMarginId, false);
+            textViewOptions.SetOptionValue(LineNumbersMarginOptions.LineNumbersMarginOptionId, true);
+
+            textViewOptions.OptionChanged += OnLineNumberOptionChangedSync;
+        }
+
+        private void DisableVimLineNumbersMargin(IEditorOptions textViewOptions)

```

Comment:
This method is never called

Reply:
I wanted to bind it to a new VimApplication setting once we merge settings cache, so users have option to opt out of this relative line numbers margin and fallback to old behavior with native line numbers margin.